package.json

```diff
@@ -97,7 +97,9 @@
     "select2": "4.0.13",
     "sortablejs": "1.14.0",
     "underscore": "1.13.1",
-    "validate.js": "0.13.1"
+    "validate.js": "0.13.1",
+    "oe-cqm-parsers": "github:openemr/oe-cqm-parsers",
+    "oe-cqm-service": "github:openemr/oe-cqm-service"
```

Comment:
This probably should be a dev dependency only. Again we don't need service in production.

Reply:
@sjpadgett This will be required to run reports. It's the eCQM service. Do you just mean the oe-cqm-parsers ?

Reply:
I unsure now. Just leave and I'll figure it out once I get to setting up for testing.